public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -550,6 +549,34 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
         });
     }
 
+    if (this.isMigrationToggleOnAndIsAccessProxy()) {
+      const target: InfluxQuery = {
+        refId: 'metricFindQuery',
+        query: 'SHOW TAG KEYS',
+        rawQuery: true,
+      };
+      return lastValueFrom(super.query({ targets: [target] } as DataQueryRequest))
+        .then((res: DataQueryResponse) => {
+          if (!res || !res.data || res.state !== LoadingState.Done) {
+            return {
+              status: 'error',
+              message: 'Error reading InfluxDB. Please make sure you have sufficient permissions.',
```

Comment:
I wonder we will be reaching even if there is connectivity issue. doesn't need to be permissions only.

Reply:
@gabor, what do you think? any thoughts?

Reply:
hmm.... :

1. i'd also just say "Error reading InfluxDB"
2. slightly below, the no-tags-found has an error-message. also there, i would go with something shorter, like "Successfully connected to InfluxDB, but no tags found"